3rdparty/sgxsdk/update.make

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/make -f
```

Comment:
Does this file need to be checked-in?  It is not used and the `wget` command would work only on Linux.

Reply:
I see that other 3rdparty directories have this file checked in, and some of them use `wget` too.